test/unit/orientedimagelayer.js

```diff
@@ -2,69 +2,94 @@ import proj4 from 'proj4';
 import assert from 'assert';
 import OrientedImageLayer from 'Layer/OrientedImageLayer';
 import OrientedImageSource from 'Source/OrientedImageSource';
-import { HttpsProxyAgent } from 'https-proxy-agent';
 import Coordinates from 'Core/Geographic/Coordinates';
 import GlobeView from 'Core/Prefab/GlobeView';
+import sinon from 'sinon';
+import Fetcher from 'Provider/Fetcher';
 import Renderer from './bootstrap';
 
+import panoData from '../data/OrientedImage/panoramicsMetaDataParis.geojson';
+import camCalibration from '../data/OrientedImage/cameraCalibration.json';
+
+const orientationsUrl = 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/immersive/exampleParis1/panoramicsMetaDataParis.geojson';
+const calibrationUrl = 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/immersive/exampleParis1/cameraCalibration.json';
```

Comment:
Maybe declare an other const for `https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/immersive/exampleParis1/`

This comment applied to multiple files

Reply:
Done. In several files.